### ISSUE TITLE
AmazMod 1.1.0 - wakelock issue

### ISSUE THREAD

**Issue by toncheee** [AmazMod/AmazMod#355]
Hi, after updating to version 1.1.0 my phone stays awake most of the time.
Culprit in BBS says that *job*/com.edotassi.amazmod/notification.NotificationJobService is the main reason my phone is awake.
What could I do to make it normal?

**Comment by fcaronte** [comment]
yes we know this are added to avoid to lost message and connection on a lot of phone, I ask to @lfom if can be put as optional connection method into the setting

**Comment by toncheee** [comment]
![screenshot_2019-01-31-19-13-55-431_com asksven betterbatterystats](https://user-images.githubusercontent.com/47208303/52075573-ec342400-258c-11e9-8857-59331e465050.png)
See my usage today...

**Comment by fcaronte** [comment]
yes, noted immediatelly this change on my phone because, yes is not to much increasing , but on my old battery phone i have a lot more phone is wake and for this i ask if possible to put as optional setting
![screenshot_20190131-191848_gsam battery monitor](https://user-images.githubusercontent.com/26366403/52075765-6664a880-258d-11e9-92cf-ca311b6bf5eb.jpg)

**Comment by fcaronte** [comment]
is not the best screen because i charge my phone from 50 to 80

**Comment by fcaronte** [comment]
This is the situation at 13.40 phone in my pocket using not to much
![screenshot_20190201-134234_gsam battery monitor](https://user-images.githubusercontent.com/26366403/52123724-41bd0f00-2627-11e9-9e43-9adacffe49f8.jpg)

![screenshot_20190201-134112_gsam battery monitor](https://user-images.githubusercontent.com/26366403/52123656-1afed880-2627-11e9-8833-fa5fca84e502.jpg)
![screenshot_20190201-134102_gsam battery monitor](https://user-images.githubusercontent.com/26366403/52123659-1b976f00-2627-11e9-830a-4f660c29d451.jpg)
![screenshot_20190201-134052_gsam battery monitor](https://user-images.githubusercontent.com/26366403/52123660-1b976f00-2627-11e9-961c-075575c63d61.jpg)

**Comment by lfom** [comment]
It is needed, it was improved in releases after 1.1.0 (dev branch).
